setup.cfg

```
@@ -23,9 +23,11 @@ exclude =
     .scratch,
     _vendor,
     data
-select = E,W,F
-ignore = W504
-
+per-file-ignores =
+    # B011: Do not call assert False since python -O removes these calls
+    tests/*: B011
+    # TODO: Remove this when fixing flake8-bugbear warnings in source
+    src/pip/*: B
```

@deveshks since the list is short, let's make it explicit.

```suggestion
    src/pip/*: B007,B008,B009,B014,B305
```

The rest LGTM.

---

@sbidoul 

I am actually going to get rid of that line in a follow-up PR, where I add `flake8-bugbear` to all files in `src/pip/*` (as specified in the TODO above) .

Given that, do I still need to make this change? If not, we can go ahead and merge this PR and I will make that follow-up PR.

---

I think an explicit list is better in the meantime, because we are then sure that no one else will introduce other warnings while you work on your pr. You can then remove the ignores one at a time.

---

That's a very valid point. I have gone ahead and incorporated that change in https://github.com/pypa/pip/pull/8398/commits/7744bd2102a416e5ddf1218a8e06b47f6cbf7648

---

The CI  is green now (The one test showing pending has actually passed, but the status hasn't been updated https://travis-ci.org/github/pypa/pip/builds/695113919?utm_source=github_status&utm_medium=notification )

I think this can be merged now